### ISSUE TITLE
Jest environment not working

### ISSUE THREAD

**Issue by Gizmo-Rishab** [eslint/eslint#12789]
**Tell us about your environment**
- "jest": true,
- "browser": true,
- "es6": true,
- "node": true

* **ESLint Version:** v6.8.0
* **Node Version:** v13.6.0 
* **npm Version:** v6.13.4

**What parser (default, Babel-ESLint, etc.) are you using?**
eslint-config-google
**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js
{
    "env": {
        "jest": true,
        "browser": true,
        "es6": true,
        "node": true
    },
    "extends": "google",
    "globals": {
        "Atomics": "readonly",
        "SharedArrayBuffer": "readonly"
    },
    "parserOptions": {
        "ecmaVersion": 2020,
        "sourceType": "script"
    },
    "ignorePatterns": [
        ".cache",
        "dist",
        "node_modules"
    ],
    "rules": {
        "indent": "off",
        "no-tabs": "off",
        "linebreak-style": [
            "error",
            "windows"
        ],
        "quotes": [
            "error",
            "double"
        ],
        "semi": "error",
        "comma-dangle": "off",
        "no-unused-vars": [
            "error",
            {
                "vars": "local",
                "args": "none"
            }
        ],
        "no-console": "warn",
        "require-jsdoc": "off",
        "object-curly-spacing": "off"
    }
}
```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```js
const greet = require("../index");

test("is greeting with user", () => {
    const response = greet("Rishab");
    expect(response).toBe("Greetings Rishab, have a nice day! :)");
});

test("is greeting without user", () => {
    const response = greet();
    expect(response).toBe("Greetings User, have a nice day! :)");
});

```

VSCode eslint plugin used

**What did you expect to happen?**
No error underlines

**What actually happened? Please include the actual, raw output from ESLint.**
The test and expect calls threw an error

**Are you willing to submit a pull request to fix this bug?**
No

**Comment by Gizmo-Rishab** [comment]
Oops wrong repo
